workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css

```diff
@@ -257,7 +257,7 @@ pre[class*='language-'] {
 				padding: 0 4px;
 			}
 		}
-
+		
```

Comment:
@rtienpal we should be careful about whitespace, we dont want to add this here. Also make sure to run both gw formatSource before sending